cmd/deploy/deploy.go

```diff
@@ -450,10 +450,13 @@ func (dc *Command) Run(ctx context.Context, deployOptions *Options) error {
 	oktetoLog.AddToBuffer(oktetoLog.InfoLevel, "EOF")
 
 	if err != nil {
-		if err == oktetoErrors.ErrIntSig {
+		if errors.Is(err, oktetoErrors.ErrIntSig) {
 			return nil
 		}
-		err = oktetoErrors.UserError{E: err}
+		// transform internal errors to user errors
```

Comment:
@ifbyol the bug was introduced here, i fixed now. I refactored this as we where transforming the err to UserError missing the hint, when i change that i returned the err instead of just overwrite it. with this last change is all working as expected